snippets/serverless-about-kafka-broker.adoc

```diff
@@ -0,0 +1,22 @@
+// Text snippet included in the following modules and assemblies:
+//
+// * /modules/serverless-broker-types.adoc
+// * /serverless/develop/serverless-kafka-developer.adoc
+
+:_content-type: SNIPPET
+
+For production-ready Knative Eventing deployments, Red Hat recommends using the Knative Kafka broker implementation. The Kafka broker is an Apache Kafka native implementation of the Knative broker, which sends CloudEvents directly to the Kafka instance.
+
+[IMPORTANT]
+====
+The Kafka broker is not supported on FIPS.
```

Comment:
Spell out first usage of FIPS?

Reply:
ACK, also slightly reworded to make more sense (reused wording added to latest release notes).